### ISSUE TITLE
out center overpass query returns history for node instead of way

### ISSUE THREAD

**Issue by danieldegroot2** [maproulette/maproulette3#2065]
**Describe the bug**
Ref https://github.com/maproulette/maproulette-backend/issues/943 .
(Seems not resubmitted on frontend due to long reply time -slightly over 6 months- or being unable to do so.)
Feel free to close this issue if it already exists elsewhere. Could not find it when trying to search for it here.

This causes a minor inconvenience where your challenge returns the correct coordinates but instead of ways, nodes are returned. For logged in users this means the correct active window (coordinates) will be displayed in editors, but it will select the wrong object.
For logged out users it will show an incorrect object link and history at the bottom, which is useless for quality assurance.

**Expected behavior**
`out center` returns node/way type correctly.

**To Reproduce**
I.e. for (`osmid: 942962735`)
https://maproulette.org/challenge/40253/task/170872758 when logged out shows/showed the object history for the node with identical id somewhere in the US. etc. (`node/942962735`)

Owner has already been notified and will use workaround given at the bottom of this post.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: Any
 - Version: Any

**Additional context**
`out center` is used to prevent getting a task for each child node of a way.

Workaround: Use `out geom`, this should give the way geometry instead of its child nodes (regular overpass) or center (here).

cc: @tordans

**Comment by danieldegroot2** [comment]
Sidenote: This challenge has since been superseded by
https://maproulette.org/browse/challenges/41015
using the workaround.

**Comment by mvexel** [comment]
Let's just add this to the documentation (should not use `out center`)

**Comment by CollinBeczak** [comment]
Resolved by this PR: https://github.com/maproulette/docs/pull/40, let me know if there are any changes needed to implementation or description of the notice.

**Comment by CollinBeczak** [comment]
Currently, out geom will still drop relations because there is nothing in place to build the geometry collection yet required for relations. But this will hopefully be addressed soon in this pr: https://github.com/maproulette/maproulette3/pull/2301

**Comment by danieldegroot2** [comment]
Looks okay.

**Comment by danieldegroot2** [comment]
Note it will only be resolved once https://github.com/maproulette/docs/pull/40 is merged
